src/main/java/com/MyApp/budgetControl/api/ExpensesController.java

```diff
@@ -24,20 +26,21 @@ public class ExpensesController {
     private final ExpensesService expensesService;
 
     @GetMapping
-    public List<Expense> getExpenses() {
-        return expensesService.getExpenses();
+    public List<ExpenseEntity> getExpenses() {
```

Comment:
Thats BIG red flag - You should never return ENTITY directly from controller, this should be DTO, entities are only for DB manipulation

Reply:
I changed it to use the new ResponseDTO